### PR TITLE
Updated base css styling of material-ui TextAreaAutoSize Component, #27

### PR DESCRIPTION
**Pull request by Dan2D**
Fixes Issue #27 
Added base styling of material-ui TextAreaAutoSizeComponent in the Resources index.js file

### CODE REVIEW

File: src/components/Resources/index.js [hunk 1]
```diff
@@ -44,7 +44,19 @@ const useStyles = makeStyles(theme => ({
   },
   avatar: {
     backgroundColor: red[500]
-  }
+  },
+  textArea: {
+    fontFamily: '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
```
Comment (Aprillion) [inline src/components/Resources/index.js]: would it be possible to use `fontFamily: inherit`?

Comment (Dan2D) [reply src/components/Resources/index.js]: Of course, I'll update it and resend the PR

Comment (Dan2D) [reply src/components/Resources/index.js]: Updated the fontFamily, let me know if anything else needs changing

Comment (Aprillion) [reply src/components/Resources/index.js]: looks good to me,, but to be clear: I'm just a random person browsing hacktoberfest, not a maintainer of this project.

Comment (lpatmo) [reply src/components/Resources/index.js]: Haha, thanks to you both, @Dan2D and @Aprillion

**Pull request merged**